### ISSUE TITLE
Major camera glitches when fastroping

### ISSUE THREAD

**Issue by alganthe** [acemod/ACE3#3459]
**Arma 3 Version:** `1.56.134627 stable`

**ACE3 Version:** `3.5.0RC1`

**Mods:**
* `@CBA_A3 2.3.1`
* `@ace`

**Placed Modules:**
None

**Description:**
Violent camera glitches happen when using fastrope, example video:

**THE VIDEO HAS RAPIDLY FLASHING IMAGES, BEWARE**
https://www.youtube.com/watch?v=L2uRXopW-jQ

(the video does present others mods, but that's because I have a shitty connection, I tested this with the mods listed above)

**Steps to reproduce:**
- Place a playable unit in the editor, give it access to zeus.
- Press play.
- Spawn a huron.
- Double click on the huron and in the execute box paste this:
`this disableAI "FSM";this disableAI "MOVE";`
- Come back to your character, aim at the helo and execute:
`player moveInCargo cursorObject`
- Place the helo at a reasonable height (it shouldn't try to move at that point)
- Deploy the ropes and enjoy the flashbacks.

**Where did the issue occur?**
SP (3DEN), MP (self hosted)

**RPT log file:**
related errors:
```
22:06:59 WARNING: SuppressCache::Trace called with huge input. Function exited to prevent program stall inside. tBeg[-86976.132813,3028.553955,26856.878906]; tEnd[-86827.218750,3037.001465,26872.794922]; ammoHit(3.000000); distance(150.000000)
22:07:00 WARNING: SuppressCache::Trace called with huge input. Function exited to prevent program stall inside. tBeg[-197170.890625,24784.263672,-1966.349609]; tEnd[-197041.843750,24797.447266,-1891.038208]; ammoHit(3.000000); distance(150.000000)
```

**Comment by jonpas** [comment]
Reproduced by @BaerMitUmlaut

**Comment by esteldunedain** [comment]
Fixed in #3461.
